cmd/kpod/images.go

```diff
@@ -155,7 +158,7 @@ func outputImages(store storage.Store, images []storage.Image, format string, ha
 			ID:        img.ID,
 			Name:      name,
 			Digest:    digest,
-			CreatedAt: createdTime,
+			CreatedAt: createdTime.Format("Jan 2, 2006 15:04"),
```

Comment:
According to the godoc for the `time` package, the values in the format string are compared with values of fields in a structure that represents a reference time value to determine which field corresponds to a given portion of the format string, and the value of the corresponding field in `createdTime` is formatted to match it to produce the result.